src/js/components/Accordion/doc.js

```diff
@@ -28,7 +28,10 @@ accordingly.`
     animate: PropTypes.bool.description(
       'Transition content in & out with a slide down animation.'
     ).defaultValue(true),
-    children: PropTypes.arrayOf(PropTypes.node).description(
+    children: PropTypes.oneOfType([
+      PropTypes.node,
+      PropTypes.arrayOf(PropTypes.node),
+    ]).description(
```

Comment:
Can this be just `PropTypes.node`? According to the prop-types documentation, a node can be an array.
>  // Anything that can be rendered: numbers, strings, elements or an array
>  // (or fragment) containing these types.